go-controller/pkg/ovn/controller/services/loadbalancer.go

```diff
@@ -136,13 +180,23 @@ func EnsureLBs(nbClient libovsdbclient.Client, service *corev1.Service, existing
 		mapLBDifferenceByKey(removeLBsFromGroups, existingGroups, wantGroups, blb)
 	}
 
-	ops, err := libovsdbops.CreateOrUpdateLoadBalancersOps(nbClient, nil, existinglbs...)
+	ops, err := libovsdbops.CreateOrUpdateLoadBalancersOps(nbClient, nil, toNBLoadBalancerList(existinglbs)...)
+	if err != nil {
+		return err
+	}
+
+	ops, err = libovsdbops.CreateLoadBalancersOps(nbClient, ops, toNBLoadBalancerList(newlbs)...)
 	if err != nil {
 		return fmt.Errorf("failed to create ops for ensuring update of service %s/%s load balancers: %w",
 			service.Namespace, service.Name, err)
 	}
 
-	ops, err = libovsdbops.CreateLoadBalancersOps(nbClient, ops, newlbs...)
+	ops, err = svcCreateOrUpdateTemplateVarOps(nbClient, ops, toNBTemplateList(newlbs))
+	if err != nil {
+		return err
+	}
+
+	ops, err = svcCreateOrUpdateTemplateVarOps(nbClient, ops, toNBTemplateList(existinglbs))
```

Comment:
nit alert

Would this be equivalent to just doing

```
ops, err = svcCreateOrUpdateTemplateVarOps(nbClient, ops, toNBTemplateList(tlbs))
```

Reply:
Way better, i'll do that!